.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: objective-c
+install:
+  - bundle install
+  - brew outdated carthage || brew upgrade carthage
+script:
+  - ./Scripts/ci.sh
+matrix:
+  include:
+  - osx_image: xcode11.2
+    env: ACTION="swift-package";SDK="iphonesimulator";DESTINATION="platform=iOS Simulator,OS=13.2.2,name=iPad Pro (12.9-inch) (3rd generation)";SHOULD_TEST="true"
+  - osx_image: xcode11.2
+    env: ACTION="swift-package";SDK="appletvsimulator13.2";DESTINATION="platform=tvOS Simulator,name=Apple TV";SHOULD_TEST="true"
+  - osx_image: xcode11.2
+    # We can't run tests on macOS because CacheAdvance supports a minimum of macOS 10.15, and Travis CI's machines run on macOS 10.14.
```

Comment:
Yes (I plan on filing a bunch of issues when I merge 😅). I'll add this to the issue, but the two options I see are:
1. Move to Circle CI (which supports Catalina)
2. Wait until travis supports Catalina (it can't be stuck on 10.14 forever).